google/cloud/security/inventory/inventory_loader.py

```diff
@@ -256,36 +249,52 @@ def main(argv):
     # Otherwise, there is a gap where the ratelimiter from one pipeline
     # is not used for the next pipeline using the same API. This could
     # lead to unnecessary quota errors.
-    crm_rate_limiter = CloudResourceManagerClient.get_rate_limiter()
-    admin_directory_rate_limiter = AdminDirectoryClient.get_rate_limiter()
+    #
+    # TODO: Move the building of the rate limiter and credential
+    # to the api client:
+    # rate limit getting should be from the module
+    # rate limit setting should be passed into the creation of the client
+    # credentials should be built inside the client and never exposed here
+    max_crm_calls = configs.get('max_crm_api_calls_per_100_seconds', 400)
+    crm_rate_limiter = RateLimiter(max_crm_calls, 100)
```

Comment:
Yes, mirror the functionality of AdminDirectoryClient.get_rate_limiter() in CloudResourceManagerClient